spec/features/seeds/adding_seeds_spec.rb

```diff
@@ -17,12 +17,23 @@
     within "form#new_seed" do
       fill_in "Quantity:", :with => 42
       fill_in "Plant before:", :with => "2014-06-15"
+      fill_in "Days until maturity, min:", :with => 999
+      fill_in "Days until maturity, max:", :with => 1999
+      select "certified organic", :from => "Organic?"
+      select "non-certified GMO-free", :from => "GMO?"
+      select "heirloom", :from => "Heirloom?"
       fill_in "Description", :with => "It's killer."
       select "internationally", :from => "Will trade:"
       click_button "Save"
     end
 
     expect(page).to have_content "Successfully added maize seed to your stash"
+    expect(page).to have_content "Quantity: 42"
+    expect(page).to have_content "Days until maturity: 999"
```

Comment:
Should this check for it displaying as a date range, since there's a partial that does different things depending on whether both min & max are filled in?